### DISCUSSION TITLE
Laravel livewire real time validation glitch with slow mobile connection

### DISCUSSION THREAD

**Discussion by Dima-Ca** [livewire/livewire#3123] [7. Bugs (archived)]
### Steps to reproduce:

Create a livewire component with this content
```

namespace App\Http\Livewire;

use Livewire\Component;

class ShowPosts extends Component
{
    protected $rules = [
        'field1' => 'required',
        'field2' => 'required',
        'field3' => 'required',
    ];

    public $field1;
    public $field2;
    public $field3;

    public function updated($propertyName)
    {
        $this->validateOnly($propertyName, $this->rules);
    }

    public function render()
    {
        return view('livewire.show-posts');
    }
}
```
And this view
```
<div>
    <form wire:submit.prevent="submit">
        <div class="form-group mb-3 col-12">
            <input class="form-control" type="text" wire:model="field1">
        </div>
        <div class="form-group mb-3 col-12">
            <input class="form-control" type="text" wire:model="field2">
        </div>
        <div class="form-group mb-3 col-12">
            <input class="form-control" type="text" wire:model="field3">
        </div>
    </form>
</div>
```

Include it in a blade view with @livewire('show-posts')
In chrome dev tools go to network->online dropdown->select slow 3g connection
Try to enter value for each input and go to next with tab, you will notice that value from previous input disappear or change to the last value that was before modification

**Context**
- Livewire version: 2.3.0
- Laravel version: 7.28.4
- Browser: Chrome

**Comment by squishythejellyfish** [comment]
👋 Oh Hi! I'm Squishy, the friendly jellyfish that manages Livewire issues.

I see this issue has been closed.

Here in the Livewire repo, we have an "issues can be closed guilt-free and without explanation" policy.

If for ANY reason you think this issue hasn't been resolved, PLEASE feel empowered to re-open it.

Re-opening actually helps us track which issues are a priority.

Reply "REOPEN" to this comment and we'll happily re-open it for you!

(More info on this philosophy here: https://twitter.com/calebporzio/status/1321864801295978497)

**Comment by Nimeshka** [comment]
I too have this issue.

**Comment by umarfarooqumzen** [comment]
REOPEN

**Comment by umarfarooqumzen** [comment]
I am facing the same issue on slow networks please can you guide how to sort it out?

**Comment by stefanfisk** [comment]
This is a major issue for me as well. My simplest idea is some sort of mechanism for ignoring validation responses for requests sent with stale data.
